ramp4-pcar4/ramp4-pcar4
# Update multi-ramp for export, correct selector

### Related Item(s)
#2476

### Changes

- [FIX] Add export to multi-ramp sample, correct selector base 

### Notes
I could not replicate the blank export download with `multi-ramp.html`. Let me know if there are configs/scenarios where it breaks multi-ramp. 

Unrelated to the bug reported but I corrected the `document` selector to use `this.$vApp.$el` instead.

### QA Testing

Please test against `main` branch https://ramp4-pcar4.github.io/ramp4-pcar4/main/demos/index-multi.html

### Testing

Steps:
1. Open [`index-multi.html`](https://milespetrov.github.io/ramp4-pcar4/2476-export/demos/index-multi.html) sample page, notice 3 ramp instances
2. Open the export panel for one or more instances
3. Download the export image, open the image and notice it is accurate


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ramp4-pcar4/ramp4-pcar4/2481)
